topics/earth-system/tutorials/ocean-data-view/tutorial.md

```diff
@@ -0,0 +1,250 @@
+
+Once you finished your work on ODV, if you want to retrieve data and/or entire project, you need to save files in /output, then quit ODV properly through the "Project" Menu tab.
+
+> <details-title>Short introduction on how Galaxy works</details-title>
+>
+> You can come back to where you left off the tutorial anytime by clicking {% icon level %}.
+>
+> > <hands-on-title>Log in to Galaxy</hands-on-title>
+> > 1. Open your favorite browser (Chrome, Safari or Firefox as your browser, not Internet Explorer!)
+> > 2. Browse to your [Galaxy instance](https://earth-system.usegalaxy.eu/)
+> > 3. On the top pannel go to **Login or Register**
+> >
+> >
+> {: .hands_on}
+>
+> The Galaxy homepage is divided into three panels:
+> * Tools on the left
+> * Viewing panel in the middle
+> * History of analysis and files on the right
+>
+> ![Screenshot of the Galaxy interface, the tools panel is on the left, the main panel is in the center, and the history is on the right.](../../images/coastal_water_dyn/galaxy_interface.png "Galaxy interface explanation")
+>
+> The first time you use Galaxy, there will be no files in your history panel.
+{: .details}
+
+
+> <hands-on-title>Deploy your own ODV instance</hands-on-title>
+>
+> 1. Create a new history for this tutorial nd give it a name (example: “ODV for Argo”) for you to find it again later if needed.
+>
+>    {% snippet faqs/galaxy/histories_create_new.md %}
+> 
+> 2. Import netCDF data from amazonaws with this link an rename it "1900392_prof.nc" instead of "New File"
+>
+>    ```
+>    https://argo-gdac-sandbox.s3-eu-west-3.amazonaws.com/pub/dac/coriolis/1900392/1900392_prof.nc
+>    ```
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}  
+>  
+> 3. Select your new data before deploying {% tool [ODV instance](https://earth-system.usegalaxy.eu/root?tool_id=interactive_tool_odv) %} by pressing **execute**
+> 4. Access ODV 
+>
+> > <tip-title> Reminder on how to acces interactive tool </tip-title>
+> >  A URL will be displayed at the top of the Galaxy center panel. If you don't see it, you can see and access it through the "Active InteractiveTools" space of the "User" menu or you can click on {% icon galaxy-eye %} on the tool in the history.
+> {: .tip}
+>
+{: .hands_on}
+
+> <details-title> Some complementary information about your data </details-title>
+> Argo is an international program to observe the interior of the ocean with a fleet of profiling floats drifting in the deep ocean currents (check it [here](https://argo.ucsd.edu)). Argo GDAC is a dataset of 5 billion in situ ocean observations from 18.000 profiling floats (4.000 active) which started 20 years ago. Argo GDAC dataset is a collection of 18.000 NetCDF files. It is a major asset for ocean and climate science, a contributor to IOCCP reports. For more informations click [here]( https://doi.org/10.17882/42182)
```

Comment:
```suggestion
> Argo is an international program to observe the interior of the ocean with a fleet of profiling floats drifting in the deep ocean currents (check it [out on the argo site](https://argo.ucsd.edu)). Argo GDAC is a dataset of 5 billion in situ ocean observations from 18.000 profiling floats (4.000 active) which started 20 years ago. Argo GDAC dataset is a collection of 18.000 NetCDF files. It is a major asset for ocean and climate science, a contributor to IOCCP reports. {% cite argodata %}
```

The 'here' link is an issue for people navigating with screenreaders, having a more descriptive link is helpful for them.

Separately we encourage that for citable objects, the DOI is added to the bibliography and referenced that way. You can add

```
@misc{argodata,
  doi = {10.17882/42182},
  url = {https://www.seanoe.org/data/00311/42182/},
  author = {{Argo}},
  keywords = {float,  Argo,  global ocean observing system,  ocean circulation,  in-situ,  ocean pressure,  sea water salinity,  sea water temperature,  multi-year,  weather climate and seasonal observation,  global-ocean},
  title = {Argo float data and metadata from Global Data Assembly Centre (Argo GDAC)},
  publisher = {SEANOE},
  year = {2023},
  copyright = {Creative Commons Attribution 4.0 International}
}
```

to the tutorial.bib file